### ISSUE TITLE
Cordova 7.0.1 empty screen

### ISSUE THREAD

**Issue by 4i-helpynet** [Le-Space/janus-echo#3]
After cloning the app (update cordova, remove android-add android) browser working perfectly, but android (with USB connection) the screen is empty - logs are too...

**Comment by inspiraluna** [comment]
Did you install and run janus?

Am 20.09.2017 16:24 schrieb "4i-helpynet" <notifications@github.com>:

**Comment by 4i-helpynet** [comment]
Yes, and browser build work too (I didn't try iOS). As I see, the problem is the video stream isn't start, even the permissions granted. I just need a working test application, I try to downgrade the build environment, didn't work... What version of cordova, android and crosswalk can I use to successfully build android app?

**Comment by inspiraluna** [comment]
I am sorry, I didn’t answer before I was on a business trip today and yesterday. I’d like to help you. I appologize for the late answer. 
I think its not very much, but the project is not updated with the latest libraries. Its almost 1 year old now. Maybe even crosswalk is not needed anymore! (I have to look into it). 

I am leaving for a 1 week holiday tomorrow. if its very urgent to you maybe you have a little budget for that, I could probably interrupt the holiday for you to fix that urgently.

Kind Regards,

Nico
